osbuild/pipeline.py

```diff
@@ -370,9 +370,12 @@ def __init__(self):
         self.pipelines = collections.OrderedDict()
         self.sources: List[Source] = []
 
-    def add_pipeline(
-            self, name: str, runner: Optional[str], build: Optional[str] = None, source_epoch: Optional[int] = None
-    ) -> Pipeline:
+    def add_pipeline(self,
+                     name: str,
+                     runner: Optional[str],
+                     build: Optional[str] = None,
+                     source_epoch: Optional[int] = None
+                     ) -> Pipeline:
```

Comment:
Can I interest anyone in this lovely (and pylint-supported) syntax:

```python
    def add_pipeline(
        self,
        name: str,
        runner: Optional[str],
        build: Optional[str] = None,
        source_epoch: Optional[int] = None
    ) -> Pipeline:
```

I will promise I will only knock on your doors promoting it once every moon-cycle!

Reply:
ok, I indulge you, but only because @achilleas-k likes it too ;)